plugins/woocommerce/changelog/50239-update-in-product-blocks-docs-links

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: tweak
+Comment: Updating documentation links, I don't believe a comment is necessary for this small change.
```

Comment:
Was this added before you checked the " This Pull Request does not require a changelog entry. (Comment required below)" box? If so this should be removed before merge? I'm not sure its supposed to create one when the box is checked.

Reply:
I'm not sure how the changelog entries get added, it looks like it was added by an automation, however I did have the  "This Pull Request does not require a changelog entry. (Comment required below)" box checked before I published this PR yesterday, so not sure why it added the entry anyway. 

Let's remove it before merging 👍 I went ahead and deleted the file.

Reply:
I asked in Slack and this is apparently expected, so leave it in place.